### PR TITLE
Pep8

### PR DESCRIPTION
**Pull request by Titan-C**
This includes the remarks for pep8 in https://github.com/nilearn/nilearn/pull/399
also the rename of make_thumbnail to scale image

### CODE REVIEW

File: sphinxgallery/__init__.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 """Sphinx Gallery
 """
 import os
-__version__ = '0.0.7'
+__version__ = '0.0.9'
```
Comment (lesteve) [inline sphinxgallery/__init__.py]: I am not sure about the origin of this diff but I can see that sphinx-gallery 0.0.8 has already been released on pypi. 

I am going to hazard a wild guess and assume that you released from a local modification which should be avoided. Tagging the release point with git is also a good idea for the future releases. I can see that you did that for a few releases in the past.

File: sphinxgallery/gen_rst.py [hunk 2]
```diff
@@ -308,9 +310,10 @@ def generate_dir_rst(directory, fhindex, examples_dir, gallery_dir, gallery_conf
     """)  # clear at the end of the section
 
 
-def make_thumbnail(in_fname, out_fname, width, height):
-    """Make a thumbnail with the same aspect ratio centered in an
-       image with a given width and height
+def scale_image(in_fname, out_fname, max_width, max_height):
+    """Scales an image with the same aspect ratio centered in an
+       image with a given max_width and max_height
+       if in_fname == out_fname the image can only be scaled down
```
Comment (lesteve) [inline sphinxgallery/gen_rst.py]: Minor style comment: the short description of a function is supposed to fit on one line, see PEP 257 and/or [numpydoc](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) convention.

File: sphinxgallery/gen_rst.py [hunk 3]
```diff
@@ -319,14 +322,17 @@ def make_thumbnail(in_fname, out_fname, width, height):
...
+    scale_h = max_height / float(height_in)
 
-    if height_in * scale_w <= height:
+    if height_in * scale_w <= max_height:
         scale = scale_w
     else:
         scale = scale_h
 
+    if scale >= 1.0 and in_fname == out_fname:
+        return
```
Comment (lesteve) [inline sphinxgallery/gen_rst.py]: Just curious, what's the purpose of this change ?

Comment (Titan-C) [inline sphinxgallery/gen_rst.py]: The change of scale_image is more an import of the mayavi branch pull request. I do find it more convenient to have it like this than the old make_thumbnail function and it is something I started working early on for this new release start.
Why this last line? It is actually the work of @agramfort

File: sphinxgallery/__init__.py (hunk 1 above)
Comment (Titan-C) [inline sphinxgallery/__init__.py]: Yes version 0.0.8 was an emergency release. Because there was a conflict in the CSS hierarchy when merging release 0.0.7 to nilearn(There is a big difference in css if you have div.class or .class I only matters if earlier in you CSS you have defined some directives like div.class, which is the case for some sphinx themes), but I could not make a release with the same version number 0.0.7 in pypi. Thus the release to pypi was a mere local change. Then I missed to update sphinx-gallery with the 0.0.8 version number and make the release.

File: sphinxgallery/gen_rst.py (hunk 3 above)
Comment (lesteve) [inline sphinxgallery/gen_rst.py]: Just a remark for the future: it's actually a good idea to keep unrelated work in separate branches. I can see that you have the same commit in different PR branches. A good rule of thumb when you start working on a new feature is to do it in a new branch you created off master.

**Comment by Titan-C** [comment]
This PR became old and irrelevant
